test/e2e/service.go

```diff
@@ -1386,7 +1386,7 @@ func startServeHostnameService(c *client.Client, ns, name string, port, replicas
 		Name:                 name,
 		Namespace:            ns,
 		PollInterval:         3 * time.Second,
-		Timeout:              30 * time.Second,
```

Comment:
Can we use the standard timeout in e2e/utils please.  It's also currently set to 2 minutes:

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/util.go#L84